website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -60,6 +61,28 @@ For example, given the forward-to URL `https://my-custom-app.com/api/123`, the `
 - `forward-url-full-path`: `https://my-custom-app.com/api/123`
 - `forward-url-origin`: `https://my-custom-app.com`
 
+## Configuring custom claims
+
+<Info>
+
+This feature is available from version `21.14.0` onwards.
+
+</Info>
+
+This header can be used to make the Merchant Center API Gateway to include extra public claims in the exchange token sent along with forwarded requests.
+
+The value must be a list of the claims separated by an empty space.
```

Comment:
Not sure about this one since we agreed this header should use an empty space as claims separator in the RFC and it's how it's been implemented in the MC Gateway.

Reply:
My bad sorry, I got confused. 🤦‍♂️